trainingMaquetado/Maquetado.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+
+<html>
+
+<head>
+   
+	<div class= "headline"></div>
+    <title>Cheese Guardian</title>
+    <link rel="stylesheet" href="Maquetado.css">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+    <!-- This is a comment, by the way -->
+
+</head>
+
+<body>
+
+
+
+<div class="upper-container">
+    <img class= "esthetic" src="photos/phone.png">
+    <hr>
+    <img  src="photos/tcg_logo.png">
+</div>
+
+<div class = "middle-container">
+	
+</div>
+
+
+<div class = "lower-container">
+</div>
+
+</body>
+
+</html>
```

Comment:
add an empty line at the end of file (check your text editor, you can add a rule that adds an empty  line at the end of the file every time you save it)